### PR TITLE
Add shinton-Lawrence file

### PR DESCRIPTION
**Pull request by shinton**
First I created a skeleton document for Lawrence, then I added a few links. You'll find my favorite 🍺 restaurant and the big school 🎓 here in town.

Any more categories or links you think I should add?

### CODE REVIEW

File: shinton-Lawrence.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Lawrence, KS
+
+### About
+
+[Wikipedia Page](https://en.wikipedia.org/wiki/Lawrence,_Kansas)
+
+### Restaurants
+
+[Free State Brewing](http://www.freestatebrewing.com)
```
Comment (tonjann) [inline shinton-Lawrence.md]: Is this your favorite brewery? @shinton

**Comment by tonjann** [comment]
Thank you for providing links! :+1:

File: shinton-Lawrence.md (hunk 1 above)
Comment (shinton) [inline shinton-Lawrence.md]: It is indeed! I'm going there tonight!

**Comment by githubteacher** [comment]
👍  this looks ready!

**Pull request merged**